clang/test/CodeGenCXX/auto-var-init-max-size.cpp

```diff
@@ -15,7 +15,7 @@ struct Foo {
 
 int foo(unsigned n) {
   bool var_size_1;
-  long var_size_8 = 123;
```

Comment:
Can we have both the test cases in which scalar variables are auto-inited and where the initialization is skipped?
Something along the lines of:

```
long var_size_8_init = 123;
long var_size_8_noinit;
long var_size_8_init_later;
...
var_size_8_init_later = 456;
```

Reply:
Sure! Added those cases.

Reply:
LGTM